keepassxc-browser/content/autocomplete.js

```diff
@@ -193,18 +199,14 @@ class Autocomplete {
                 this.selectItem();
             }
         } else if (e.key === 'ArrowUp' && this.list) {
-            e.preventDefault();
+            cancelEvent(e);
             const items = this.getAllItems();
             this.index = (this.index > 0 ? this.index : items.length) - 1;
             this.selectItem();
         } else if (e.key === 'Enter' && this.input) {
-            if (inputField.value === '') {
-                e.preventDefault();
-            }
```

Comment:
I don't think this is necessary anymore. Not sure what the purpose was actually. We check below if an item is selected and if so we call `preventDefault`. I don't think we have to do it when an item isn't selected.

Reply:
If I remember correctly, this prevented the submit when Autocomplete Menu was active and enter was pressed, but the field was still empty. Maybe. Remember to comment your code people.. :)